iceoryx2/src/service/static_config/message_type_details.rs

```diff
@@ -120,3 +122,124 @@ impl MessageTypeDetails {
             && self.payload.alignment <= rhs.payload.alignment
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use iceoryx2_bb_testing::assert_that;
+
+    #[test]
+    fn test_from() {
+        struct MyPayload {
+            _a: i32,
+            _b: bool,
+            _c: i64,
+        }
+
+        let got = MessageTypeDetails::from::<i32, i64, MyPayload>(TypeVariant::FixedSize);
```

Comment:
Could, you please name it `sut` instead of `got`. SUT = System Under Test. And `want` to `expected`. This is how we usually name the variables in tests - then it would be more consistent.

In my opinion it also reads better:

```rust
assert_that!(sut, eq expected);
```